runtime/devnet/src/config/contracts.rs

```diff
@@ -78,3 +78,17 @@ impl pallet_contracts::Config for Runtime {
 	type WeightPrice = pallet_transaction_payment::Pallet<Self>;
 	type Xcm = pallet_xcm::Pallet<Self>;
 }
+
+// IMPORTANT: only runtime calls through the api are allowed.
+#[test]
+fn contracts_prevents_runtime_calls() {
```

Comment:
Thanks. Not sure if need to be so verbose tho. The below is just as easy to understand and is only 4 lines.

```
assert_eq!(
  TypeId::of::<<Runtime as pallet_contracts::Config>::CallFilter>(), 
  TypeId::of::<Nothing>()
);
```

Reply:
Refactored. Added one test that we already had to testnet, sorry for poluting the PR.